src/Cake.NuGet/Install/NuGetPackageInstaller.cs

```diff
@@ -59,9 +59,17 @@ internal sealed class NuGetPackageInstaller : INuGetPackageInstaller
             _config = config ?? throw new ArgumentNullException(nameof(config));
             _currentFramework = NuGetFramework.Parse(_environment.Runtime.TargetFramework.FullName, DefaultFrameworkNameProvider.Instance);
             _nugetLogger = new NuGetLogger(_log);
+
+            var nugetConfig = GetNuGetConfigPath(_environment, _config);
+
+            var nugetConfigDirectoryPath = nugetConfig.Item1;
+            var nugetConfigFilePath = nugetConfig.Item2;
+
+            _nugetLogger.LogDebug($"Found NuGet.config at: {nugetConfigFilePath}");
```

Comment:
Use `_log` Instead. `_nugetLogger` is just for NuGet.Client internals.

Reply:
Alright 👍 Done